src/api/external-system.ts

```diff
@@ -29,13 +35,19 @@ export interface ExternalSystemInput {
   type: ExternalSystemType;
   username: string;
   apiKey: string;
+  operationMode: VanOperationMode;
 }
 
 export const schema = `
   enum ExternalSystemType {
     VAN
   }
 
+  enum VanOperationMode {
+    Voterfile
+    MyCampaign
+  }
```

Comment:
I think it's good to have an operation mode resolver / graphql field that is just derived / stored from the `|0` or `|1` in the api key. Otherwise, there's only a chance they can get out of sync if that info is stored in two places

Reply:
👍  makes sense